app/assets/javascripts/user.js

```diff
@@ -124,4 +124,9 @@ $(document).ready(function () {
     $("#contributorTerms").html("<img src='" + OSM.SEARCHING + "' />");
     $("#contributorTerms").load(url);
   });
+
+  $("#read_tou").on("click", function () { 
+    var agreebtn = document.getElementById('agree');
+    agreebtn.disabled=!this.checked;
```

Comment:
I'd rather do this with jQuery like everything else - something like this I think:

```
$("#agree").prop("disabled", !$(this).prop("checked"))
```